server/src/main/java/org/cloudfoundry/identity/uaa/provider/JdbcIdentityProviderProvisioning.java

```diff
@@ -56,6 +61,24 @@ public JdbcIdentityProviderProvisioning(JdbcTemplate jdbcTemplate) {
         this.jdbcTemplate = jdbcTemplate;
     }
 
+    public boolean idpWithAliasExistsInZone(final String zoneId) {
+        final Integer idpWithAliasExists;
+        try {
+            idpWithAliasExists = jdbcTemplate.queryForObject(
```

Comment:
please return from here, e.g. 

return jdbcTemplate.queryForObject( xxx ) == 1;

and omit the parts below, which are not covered by tests yet

Reply:
fixed with commit a09808b